lib/api/models/request.js

```diff
@@ -21,7 +19,7 @@ export class Request {
   buildMockedRequestData() {
 
     const defaultState = {
-      name: 'Original Response',
+      name: 'Mocked Response',
```

Comment:
It's very confusing now. Because when I see 'original response' in the UI, I immediately assume that it is the original response and not the mocked one.....

Maybe we can call it "mocked (original)", or something like that?